pkg/services/ngalert/schedule/alert_rule.go

```diff
@@ -227,10 +227,16 @@ func (a *alertRule) Run(key ngmodels.AlertRuleKey) error {
 			}
 
 			func() {
+				orgID := fmt.Sprint(key.OrgID)
```

Comment:
This is a small nit, but I prefer to use `strconv.Itoa`. It's on average twice as fast as `fmt.Sprint`, and is almost allocation-free for small numbers as it uses a lookup table https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/strconv/itoa.go;l=58-81.

Reply:
Fair point - TBH I just moved the code from [lower down the file](https://github.com/grafana/grafana/pull/85873/files#diff-4b931e3d763eee3b0d2027bfcd27ecc38478541f333322a25d59225cd3c80152R324-L316)